src/App.tsx

```diff
@@ -1,10 +1,60 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useContext, useEffect, useMemo, useState,
+} from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import { Header } from './components/Header';
+import { getTodos } from './api/todos';
+import { Footer } from './components/Footer';
+import { FilterValues } from './types/FilterValues';
+import { ErrorValues } from './types/ErrorValues';
+import { Notification } from './components/Notification';
+import { ErrorContext } from './context/ErrorContextProvider';
+
+const USER_ID = 10544;
 
 export const App: React.FC = () => {
+  const [todosFromServer, setTodosFromServer] = useState<Todo[] | null>(null);
+  const [filterValue, setFilterValue] = useState(FilterValues.All);
+
+  const filteringTodos = (todos: Todo[] | null, value: FilterValues) => {
+    if (!todos) {
+      return todos;
+    }
```

Comment:
What if there are no todos? How would you map filteringTodos?
By the way, rename to "filteredTodos"